packages/ui/src/components/editor/properties/envmap/index.tsx

```diff
@@ -164,7 +164,6 @@ export const EnvMapEditor: EditorComponentType = (props) => {
           max={10}
           value={envmapComponent.envMapIntensity.value}
           onChange={updateProperty(EnvmapComponent, 'envMapIntensity')}
-          onRelease={commitProperty(EnvmapComponent, 'envMapIntensity')}
```

Comment:
why was the commitProperty removed? how does the value propagate to the authoring layer now?